tokio/src/sync/semaphore.rs

```diff
@@ -282,7 +282,7 @@ impl Semaphore {
     }
 
     /// Returns true if the semaphore is closed
-    pub fn is_closed(&self) {
+    pub fn is_closed(&self) -> bool {
```

Comment:
Oh my... :facepalm: 

I should've run the tests prior submitting this. :see_no_evil: